aries_vcx/src/handlers/issuance/credential_def/revocation_registry.rs

```diff
@@ -0,0 +1,61 @@
+use crate::error::{VcxErrorKind, VcxResult};
+use crate::handlers::issuance::credential_def::PublicEntityStateType;
+use crate::libindy::utils::anoncreds;
+use crate::libindy::utils::anoncreds::RevocationRegistryDefinition;
+
+#[derive(Clone, Deserialize, Debug, Serialize, PartialEq)]
+pub struct RevocationRegistry {
+    pub rev_reg_id: String,
+    pub(super) rev_reg_def: RevocationRegistryDefinition,
+    pub(super) rev_reg_entry: String,
+    pub(super) tails_dir: String,
+    pub(super) max_creds: u32,
+    pub(super) tag: u32,
+    state_rev_reg_def: PublicEntityStateType,
+    state_rev_reg_delta: PublicEntityStateType,
```

Comment:
`rev_reg_def_state` and `rev_reg_delta_state` is more natural order of the words which makes it easier to understand the meaning.